gamestonk_terminal/config_terminal.py

```diff
@@ -9,7 +9,7 @@
 DEFAULT_CONTEXT = ""
 
 # By default the jupyter notebook will be run on port 8888
-PAPERMILL_NOTEBOOK_REPORT_PORT = "8888"
+PAPERMILL_NOTEBOOK_REPORT_PORT = "8892"
```

Comment:
Is your Jupyter running on 8892? The default is 8888.

Reply:
Yeah.  8888 doesn't work for me for some reason.  Will revert